app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> None:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> None:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> str:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other(int, float)):
+            return Distance(self.km + other.km)
+        else:
+            raise TypeError("Unsupported operand type for +")
+
+    def __iadd__(self, other: int | float) -> None:
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        else:
+            raise TypeError("Unsupported operand type for +")
+
+    def __mul__(self, other: int | float) -> None:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> None:
+        return Distance(self.km / other)
+
+    def __lt__(self, other: int | float) -> None:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        else:
```

Comment:
There is no need for `else`, code will not execute after return. Make changes where applicable.